src/igvfd/commands/make_audit_docstring_json.py

```diff
@@ -48,12 +48,44 @@ def get_audit_function_names_from_module(module):
     return audit_function_names
 
 
+def parse_string_to_dictionary(docstring):
```

Comment:
I think it would be good to add a docstring to this function, detailing what the expectations of the input docstring are.

Reply:
Added